docs/section10/10-4.md

```diff
@@ -0,0 +1,64 @@
+# ASHVAC – Rule 10-4
+
+**Rule ID:** 10-4
+ 
+**Rule Description:** Baseline HVAC systems using fossil fuel shall be modeled using natural gas. Except where natural gas is not available for the proposed building site, propane shall be used as the heating fuel. 
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:**  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. Applicable to baseline air-side systems 3 and 9.
+
+**Function Call:** 
+
+1. get_baseline_system_types
+2. get_heating_fuels_p
+
+
+**Manual Check:** None  
+ 
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Get list of P_RMR heating_fuels: `heating_fuels_p = get_heating_fuels_p(P_RMR)`
+- Check if other or propane are in the P_RMR: `if "PROPANE" in heating_fuels_p or "FUEL_OIL" in heating_fuels_p or "OTHER" in heating_fuels_p: oil_prop_other_p = TRUE`
```

Comment:
@claperle and @mkarpman this could instead be if heating fuel is not natural gas. A proposed building could have natural gas + propane, but the baseline in that case should use natural gas and not propane- correct?
I believe here we're assuming if a site has ONLY electricity, then natural gas is not available- correct?